tests/examples/opinions/united_states/ca7_example.html

```diff
@@ -1,336 +1,391 @@
-      <TD><A
-      HREF="http://media.ca7.uscourts.gov/cgi-bin/rssExec.pl?Submit=Display&Path=Y2013/D02-28/C:12-1751:J:Tinder:aut:T:fnOp:N:1091402:S:0">
-          Final<BR>&nbsp;Opinion</A>
-      </TD>
-      <TD><FONT SIZE="-1">Tinder</FONT>
-      </TD>
-   </TR>
-<TR></TR><TR></TR>
-   <TR BGCOLOR="#FFF0C8">
-      <TD>12-2300      </TD>
-      <TD>USA v. <BR>&nbsp; Dwayne Swims      </TD>
-      <TD>criminal      </TD>
-      <TD>02/28/2013      </TD>
-      <TD><A
-      HREF="http://media.ca7.uscourts.gov/cgi-bin/rssExec.pl?Submit=Display&Path=Y2013/D02-28/C:12-2300:J:PerCuriam:aut:T:npDp:N:1091611:S:0">
-          Nonprecedential<BR>&nbsp;Disposition</A>
-      </TD>
-      <TD><FONT SIZE="-1">PerCuriam</FONT>
-      </TD>
-   </TR>
-<TR></TR><TR></TR>
-   <TR BGCOLOR="#FFF0C8">
-      <TD>12-2313      </TD>
-      <TD>Farrokh Yassan v. <BR>&nbsp; J.P. Morgan Chase and Company      </TD>
-      <TD>civil      </TD>
-      <TD>02/28/2013      </TD>
-      <TD><A
-      HREF="http://media.ca7.uscourts.gov/cgi-bin/rssExec.pl?Submit=Display&Path=Y2013/D02-28/C:12-2313:J:Tinder:aut:T:fnOp:N:1091436:S:0">
-          Final<BR>&nbsp;Opinion</A>
-      </TD>
-      <TD><FONT SIZE="-1">Tinder</FONT>
-      </TD>
-   </TR>
-   <TR BGCOLOR="#FFF0C8">
-      <TD>12-3239      </TD>
-      <TD>Buddha Entertainment, LLC v. <BR>&nbsp; Gus A. Paloian      </TD>
-      <TD>bankruptcy from<BR>&nbsp;district court      </TD>
-      <TD>02/28/2013      </TD>
-      <TD><A
-      HREF="http://media.ca7.uscourts.gov/cgi-bin/rssExec.pl?Submit=Display&Path=Y2013/D02-28/C:12-3239:J:Easterbrook:aut:T:fnOp:N:1091350:S:0">
-          Final<BR>&nbsp;Opinion</A>
-      </TD>
-      <TD><FONT SIZE="-1">Easterbrook</FONT>
-      </TD>
-   </TR>
-</TABLE>
-</TD></TR></TABLE>
-<FONT FACE="Arial,Helvetica,sans-serif" SIZE="-2">
-   (23 documents found)
-</FONT>
-<HR>
-<FORM ACTION="http://media.ca7.uscourts.gov/cgi-bin/rssExec.pl"
-        METHOD="GET">
-   <BUTTON TYPE="SUBMIT" NAME="Submit" VALUE="Help3">Help</BUTTON>
-</FORM>
-</BODY>
-</HTML>
+<?xml version="1.0" encoding="UTF-8"?>
+<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom">
+<channel>
+<title>US 7th Judicial Circuit: New Opinions</title>
+<atom:link href="https://media.ca7.uscourts.gov/cgi-bin/OpinionsWeb/processWebInputExternal.pl?Time=month&amp;startDate=&amp;endDate=&amp;Author=any&amp;AuthorName=&amp;Case=any&amp;CaseYear=&amp;CaseNum=&amp;Rubmit=RssRecent&amp;RssJudgeName=Sykes&amp;OpsOnly=yes" rel="self" stype="application/rss+xml"/>
+<link>/opinionTest.html</link>
+<image>
+<url>http://www.ca7.uscourts.gov/ca7-itunes.jpg</url>
+<title>US 7th Judicial Circuit: New Opinions</title>
+<link>/opinionTest.html</link>
+</image>
+<description>The latest opinions released by the US 7th Circuit Court of Appeals</description>
+<language>en-us</language>
+<pubDate>
+Tue, 20 Feb 2007 10:05:00 CDT
+</pubDate>
+<lastBuildDate>
+Fri, 17 Sep 2021 11:36:00 CST
+</lastBuildDate>
+
+<item>
```

Comment:
Let's trim this file so it's much shorter (and thus faster). Maybe only five entries, or something?

Reply:
√